nginx/Dockerfile

```diff
@@ -38,11 +36,8 @@ RUN apt-get update \
     && cd .. \
     && rm -f nginx-${NGINX_VERSION}.tar.gz \
     && rm -rf nginx-${NGINX_VERSION} \
+    && apk del .build-deps \
```

Comment:
TIL. What a nice feature.

Reply:
:D I thought that was neat too! I saw it in the Dockerfile for the base image.